scheds/rust/scx_bpfland/src/stats.rs

```diff
@@ -39,12 +41,13 @@ impl Metrics {
     fn format<W: Write>(&self, w: &mut W) -> Result<()> {
         writeln!(
             w,
-            "[{}] tasks -> run: {:>2}/{:<2} int: {:<2} wait: {:<4} | nvcsw: {:<4} | dispatch -> kth: {:<5} dir: {:<5} pri: {:<5} shr: {:<5}",
+            "[{}] tasks -> r: {:>2}/{:<2} i: {:<2} pw: {:<4} w: {:<4} | nvcsw: {:<4} | dispatch -> k: {:<5} d: {:<5} p: {:<5} s: {:<5}",
```

Comment:
Maybe have multiple lines to keep it more readable?

Reply:
yeah... I was aiming to keep the output on a single line for easier parsing, but I'm not sure if that's relevant anymore, considering that we don't print the output to the logs by default now. I'll probably split that to multiple lines in a separate PR, thanks!